tsl/test/expected/tsl_tables.out

```diff
@@ -168,6 +168,13 @@ WARNING:  could not add drop_chunks policy due to existing policy on hypertable
                      -1
 (1 row)
 
+select add_drop_chunks_policy('test_table', INTERVAL '3 days', if_not_exists => true, cascade_to_materializations => true);
+WARNING:  could not add drop_chunks policy due to existing policy on hypertable with different arguments
+ add_drop_chunks_policy 
+------------------------
+                     -1
+(1 row)
+
```

Comment:
could we add a `select * from _timescaledb_config.bgw_policy_drop_chunks;` here. I don't see any other tests that show the column with a value of 'true'. Would be nice to avoid the same class of error we saw before.